### PR TITLE
2023.9: Beta release notes

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2023-09-06-release-20239.markdown [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+Enjoy the (beta) release!
+
+../Frenck
+
+<!--more-->
+
+- [Lorem ipsum!](#lorem-ipsum)
+- [Other noteworthy changes](#other-noteworthy-changes)
+- [To consider:](#to-consider)
+- [New Integrations](#new-integrations)
```
Comment (c0ffeeca7) [inline source/_posts/2023-09-06-release-20239.markdown]:
```suggestion
- [New integrations](#new-integrations)
```

File: source/_posts/2023-09-06-release-20239.markdown [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+There are many more improvements in this release; here are some of the other
+noteworthy changes this release:
+
+- [Lorem] now has support for ipsums, thanks [@frenck]!
+
+[@frenck]: https://github.com/frenck
+[Lorem]: /integrations/lorem
+
+## New Integrations
```
Comment (c0ffeeca7) [inline source/_posts/2023-09-06-release-20239.markdown]:
```suggestion
## New integrations
```

File: source/_posts/2023-09-06-release-20239.markdown [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+Found a bug or issue? Please report it in our [issue tracker](https://github.com/home-assistant/core/issues),
+to get it fixed! Or, check [our help page](/help) for guidance for more
+places you can go.
+
+Are you more into email? [Sign-up for our Building the Open Home Newsletter](/newsletter)
+to get the latest news about features, things happening in our community and
+other news about building an Open Home; straight into your inbox.
+
+## Breaking Changes
```
Comment (c0ffeeca7) [inline source/_posts/2023-09-06-release-20239.markdown]:
```suggestion
## Breaking changes
```

File: source/_posts/2023-09-06-release-20239.markdown [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+{% details "TODO" %}
+
+**Note**: Breaking changes processed up to PR X
+
+{% enddetails %}
+
+{% details "Lorem" %}
+
+The lorem lost its ipsum, replaced by dolor sit amet.
```
Comment (c0ffeeca7) [inline source/_posts/2023-09-06-release-20239.markdown]: :-)

File: source/changelogs/core-2023.9.markdown [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+title: Full Changelog for Home Assistant Core 2023.9
```
Comment (c0ffeeca7) [inline source/changelogs/core-2023.9.markdown]:
```suggestion
title: Full changelog for Home Assistant Core 2023.9
```

File: source/changelogs/core-2023.9.markdown [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+description: Detailed changelog for the Home Assistant Core 2023.9 release
+replace_regex: \s\(\[?[a-z0-9\-\s_]+\]?\)$
+---
+
+These are all the changes included in the Home Assistant Core 2023.9 release.
+
+For a summary in a more readable format:
+[Release notes blog for this release](/blog/2023/09/06/release-20238/).
+
+## Full Changelog
```
Comment (c0ffeeca7) [inline source/changelogs/core-2023.9.markdown]:
```suggestion
## Full changelog
```

**Review by c0ffeeca7 — COMMENTED** [review]
Lorem your ipsums

File: source/changelogs/core-2023.9.markdown [hunk 7]
```diff
@@ -0,0 +1,13 @@
+---
+title: Full xhangelog for Home Assistant Core 2023.9
```
Comment (Nardol) [inline source/changelogs/core-2023.9.markdown]:
```suggestion
title: Full changelog for Home Assistant Core 2023.9
```

File: source/changelogs/core-2023.9.markdown [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+description: Detailed changelog for the Home Assistant Core 2023.9 release
+replace_regex: \s\(\[?[a-z0-9\-\s_]+\]?\)$
+---
+
+These are all the changes included in the Home Assistant Core 2023.9 release.
+
+For a summary in a more readable format:
+[Release notes blog for this release](/blog/2023/09/06/release-20238/).
```
Comment (mikey0000) [inline source/changelogs/core-2023.9.markdown]:
```suggestion
[Release notes blog for this release](/blog/2023/09/06/release-20239/).
```

File: source/_posts/2023-09-06-release-20239.markdown [hunk 9]
```diff
@@ -0,0 +1,384 @@
+---
+layout: post
+title: "2023.9: Beta release notes"
+description: "The beta release notes for Home Assistant 2023.9"
+date: 2023-08-02 00:00:00
```
Comment (Nardol) [inline source/_posts/2023-09-06-release-20239.markdown]:
```suggestion
date: 2023-09-06 00:00:00
```

File: source/_posts/2023-09-06-release-20239.markdown [hunk 10]
```diff
@@ -0,0 +1,485 @@
...
+
+([@gjohansson-ST] - [#71432]) ([documentation](/integrations/trafikverket_train))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#71432]: https://github.com/home-assistant/core/pull/71432
+
+{% enddetails %}
+
+{% details "Z-Wave" %}
+
```
Comment (Quentame) [inline source/_posts/2023-09-06-release-20239.markdown:459]: Should we add that the Z-Wave server should be updated ?

I bumped core to beta and then I got the repair message about that.

Sorry haven't found where the merge was made for https://rc.home-assistant.io/latest-release-notes/

**Pull request closed without merging**